File: BaekjoonOnlineJudge/6841/6841.js
const fs = require('fs');

const input = fs.readFileSync(0, 'utf8').trim().split('\n');

const dict = {
  'CU': 'see you',
  ':-)': "I’m happy",
  ':-(': "I’m unhappy",
  ';-)': 'wink',
  ':-P': 'stick out my tongue',
  '(~.~)': 'sleepy',
  'TA': 'totally awesome',
  'CCC': 'Canadian Computing Competition',
  'CUZ': 'because',
  'TY': 'thank-you',
  'YW': "you’re welcome",
  'TTYL': 'talk to you later',
};

const result = [];

for (const line of input) {
  const word = line.trim();

  result.push(dict[word] ?? word);

  if (word === 'TTYL') break;
}

console.log(result.join('\n'));

File: BaekjoonOnlineJudge/6841/6841.kt
fun main() {
    val dict = mapOf(
        "CU" to "see you",
        ":-)" to "I’m happy",
        ":-(" to "I’m unhappy",
        ";-)" to "wink",
        ":-P" to "stick out my tongue",
        "(~.~)" to "sleepy",
        "TA" to "totally awesome",
        "CCC" to "Canadian Computing Competition",
        "CUZ" to "because",
        "TY" to "thank-you",
        "YW" to "you’re welcome",
        "TTYL" to "talk to you later"
    )

    while (true) {
        val word = readln()

        println(dict[word] ?: word)

        if (word == "TTYL") break
    }
}
